source/system/String.ooc

```diff
@@ -107,7 +107,11 @@ String: class {
 		match (this size) {
 			case 0 => this
 			case 1 => this toUpper()
-			case => this[0] toUpper() + this substring(1)
+			case =>
+				end := this substring(1)
+				result := this[0] toUpper() + end
+				end free()
+				result
```

Comment:
I think this function is more complicated with this `case` than it should be. Maybe:
```ooc
capitalize: func -> This {
    result := this clone()
    if (result size > 0)
        result[0] = result[0] toUpper()
    result
}
```
?